client/dfclient/src/resources.cpp

```diff
@@ -7,4 +7,9 @@ void Resources::UpdateTweens()
 		if (_tweens[i].progress() == 1.f)
 			_tweens.erase(_tweens.begin() + i);
 	}
+	for (auto it=_mobTweens.begin(); it!=_mobTweens.end(); ++it) {
```

Comment:
Nit: spaces around the equality and comparison operators

Reply:
Apart from that, you can just do a range for: `for (auto& p : _mobTweens) { ... }`